packages/@aws-cdk/aws-redshift/test/cluster.test.ts

```diff
@@ -0,0 +1,305 @@
+import { expect as cdkExpect, haveResource, ResourcePart } from '@aws-cdk/assert';
+import '@aws-cdk/assert/jest';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as kms from '@aws-cdk/aws-kms';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as cdk from '@aws-cdk/core';
+
+import { Cluster, ClusterParameterGroup, ClusterType, NodeType } from '../lib';
+
+test('check that instantiation works', () => {
+  // GIVEN
+  const stack = testStack();
+  const vpc = new ec2.Vpc(stack, 'VPC');
+
+  // WHEN
+  new Cluster(stack, 'Redshift', {
+    masterUser: {
+      masterUsername: 'admin',
+      masterPassword: cdk.SecretValue.plainText('tooshort'),
+    },
+    vpc,
+  });
+
+  // THEN
+  cdkExpect(stack).to(haveResource('AWS::Redshift::Cluster', {
```

Comment:
Remember, you can use `haveResourceLike` to only compare the properties you're interested in.

Reply:
Do you mean to say you want me to change it? Or is it a reminder for the next PR? :)

Reply:
I just want to make sure you're aware of it. I'll definitely _not_ block this PR on it :)